packages/by-name/kata/kata-runtime/0003-runtime-agent-verify-the-agent-policy-hash.patch

```diff
@@ -591,27 +590,28 @@ index 6b2b6b02d..bcb6cd2ad 100644
  		config.Bios = object.File
  	case SecExecGuest:
  		objectParams = append(objectParams, string(object.Type))
-@@ -435,6 +443,10 @@ type SocketAddress struct {
+@@ -435,6 +443,7 @@ type SocketAddress struct {
  type TdxQomObject struct {
  	QomType               string        `json:"qom-type"`
  	Id                    string        `json:"id"`
 +	SeptVEDisable         *bool         `json:"sept-ve-disable,omitempty"`
-+	MRConfigID            string        `json:"mrconfigid,omitempty"`
-+	MROwner               string        `json:"mrowner,omitempty"`
-+	MROwnerConfig         string        `json:"mrownerconfig,omitempty"`
- 	QuoteGenerationSocket SocketAddress `json:"quote-generation-socket"`
- 	Debug                 *bool         `json:"debug,omitempty"`
- }
-@@ -463,10 +475,19 @@ func (this *TdxQomObject) String() string {
+ 	MrConfigId            string        `json:"mrconfigid,omitempty"`
+ 	MrOwner               string        `json:"mrowner,omitempty"`
+ 	MrOwnerConfig         string        `json:"mrownerconfig,omitempty"`
+@@ -466,9 +475,11 @@ func (this *TdxQomObject) String() string {
  
- func prepareObjectWithTdxQgs(object Object) string {
+ func prepareTDXObject(object Object) string {
  	qgsSocket := SocketAddress{"vsock", fmt.Sprint(VsockHostCid), fmt.Sprint(object.QgsPort)}
--	tdxObject := TdxQomObject{string(object.Type), object.ID, qgsSocket, nil}
-+	tdxObject := TdxQomObject{
-+		QomType:               string(object.Type),
-+		Id:                    object.ID,
-+		QuoteGenerationSocket: qgsSocket,
-+	}
++	septVEDisable := false
```

Comment:
AFAICT we didn't set this before, did we? Is there a reason for setting it to `false`? IIRC `true` is more secure.

Reply:
The field wasn't initialized previously, and the default value for a bool in Go is false.

Reply:
It was (and still is) a pointer though, so the default should have been `nil` or am I missing something?

Reply:
I guess the question is rather what the qemu default is? 

Regarding the desired value, I understand that Linux wants to disable it: https://lkml.org/lkml/2024/8/28/510.